### PR TITLE
Fishing map/viirs match timestamps

### PR DESCRIPTION
**Pull request by j8seangel**
Add interactive timestamps for viirs match detections:
- Fit to first and last when multiple
- Fit to entire day when single

image

### CODE REVIEW

File: apps/fishing-map/features/map/popups/VesselsTable.tsx [hunk 1]
```diff
@@ -197,11 +219,62 @@ function VesselsTable({
...
+                    })}
+                  >
                     <I18nNumber number={vessel[vesselProperty]} />
+                    {hasDetectionsTimestamps && (
+                      <Fragment>
+                        {' '}
+                        (
+                        {hasMultipleDetectionsTimestamps ? (
+                          <Tooltip
+                            content={t('timebar.fitOnThisDates', 'Fit time range to this dates')}
```
Comment (nerik) [inline apps/fishing-map/features/map/popups/VesselsTable.tsx:240]:
```suggestion
                            content={t('timebar.fitOnThisDates', 'Fit time range to these dates')}
```

File: apps/fishing-map/public/locales/source/translations.json [hunk 2]
```diff
@@ -348,6 +351,8 @@
     "selected": "selected"
   },
   "timebar": {
+    "fitOnThisDates": "Fit time range to this dates",
```
Comment (nerik) [inline apps/fishing-map/public/locales/source/translations.json:354]:
```suggestion
    "fitOnThisDates": "Fit time range to these dates",
```

File: apps/fishing-map/features/map/controls/MapInfo.tsx [hunk 3]
```diff
@@ -17,14 +17,29 @@ export const pickDateFormatByRange = (start: string, end: string): DateTimeForma
   return timeΔ <= A_DAY ? DateTime.DATETIME_MED : DateTime.DATE_MED
 }
 
+export const TimeRangeDates = ({
```
Comment (nerik) [inline apps/fishing-map/features/map/controls/MapInfo.tsx:20]: Is that component used somewhere else?

**Review by nerik — APPROVED** [review]
LGTM 🚀

File: apps/fishing-map/public/locales/source/translations.json (hunk 2 above)
Comment (j8seangel) [reply apps/fishing-map/public/locales/source/translations.json]: Thanks! ✅

**Pull request merged**